internals/cli/cmd_signal.go

```diff
@@ -28,7 +28,7 @@ const cmdSignalDescription = `
 The signal command sends a signal to one or more running services. The signal
 name must be uppercase, for example:
 
-pebble signal HUP mysql nginx
+	{{.ProgramName}} signal HUP mysql nginx
```

Comment:
Any reason this should be indented?

Reply:
It's indented everywhere else (: Also, the pattern is to even include the shell caret (`$`) which I'm not sure if I should include here as well for consistency.

Reply:
The other example (in `cmd_exec.go`) doesn't use indentation or the `$ ` prefix. How about we drop them here and in `cmd_run.go` for consistency?

```
$ rg '^\s*pebble ' -C2
cmd_signal.go
29-name must be uppercase, for example:
30-
31:pebble signal HUP mysql nginx
32-`
33-

cmd_exec.go
40-arguments using "--", for example:
41-
42:pebble exec --timeout 10s -- echo -n foo bar
43-`
44-
```